components/patient-data/impl/src/main/java/org/phenotips/data/internal/controller/VariantListController.java

```diff
@@ -142,6 +159,11 @@
 
     private static final List<String> SANGER_VALUES = Arrays.asList("positive", "negative");
 
+    private static final List<String> CHROMOSOME_VALUES = Arrays.asList("1", "2", "3", "4", "5", "6", "7", "8", "9",
```

Comment:
All property values should be read from the XClass, not duplicated in the Java class. Since existing code has exactly the same problem, this doesn't have to be fixed for this PR. But it looks like a refactoring will be needed in the future.